presto-spi/src/main/java/com/facebook/presto/spi/StandardWarningCode.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi;
+
+public enum StandardWarningCode
+        implements WarningCodeSupplier
+{
+    TOO_MANY_STAGES(0x0000_0001),
```

Comment:
@martint Do we want to bother with numeric codes?

Reply:
Integer codes are already part of the API and are needed for JDBI which only has integer codes

Reply:
Do you think it would be better if the warning code name matched the error code name: QUERY_HAS_TOO_MANY_STAGES? 
For now leaving as is.